### PR TITLE
Fix output comparison for cmd tests

### PR DESCRIPTION
**Pull request by surajssd**
Now check on true as comparison output, then PASS, if anything else just fail on it. This avoids all the
false positives.

### CODE REVIEW

File: script/test/cmd/lib.sh [hunk 1]
```diff
@@ -69,8 +69,8 @@ function convert::match_output() {
 
     match=$(jq --argfile a $TEMP_STDOUT --argfile b $expected_output -n 'def post_recurse(f): def r: (f | select(. != null) | r), .; r; def post_recurse: post_recurse(.[]?); ($a | (post_recurse | arrays) |= sort) as $a | ($b | (post_recurse | arrays) |= sort) as $b | $a == $b')
 
-    if [ "$match" = false ]; then FAIL_MSGS=$FAIL_MSGS"converted output does not match\n"; return 1;
-    else SUCCESS_MSGS=$SUCCESS_MSGS"converted output matches\n"; return 0; fi
+    if [ "$match" = true ]; then SUCCESS_MSGS=$SUCCESS_MSGS"converted output matches\n"; return 0;
```
Comment (surajssd) [inline script/test/cmd/lib.sh]: Default to check if the output is true, everything else will act as fail!

File: script/test/cmd/tests.sh [hunk 2]
```diff
@@ -18,13 +18,6 @@ convert::expect_success_and_warning "kompose convert --stdout --dc -f $KOMPOSE_R
 unset $(cat $KOMPOSE_ROOT/script/test/fixtures/etherpad/envs | cut -d'=' -f1)
 
 ######
-# Tests related to docker-compose file in script/test/fixtures/flask-redis
-# kubernetes test
-convert::expect_success_and_warning "kompose convert --stdout -f $KOMPOSE_ROOT/script/test/fixtures/flask-redis/docker-compose.yml" "$KOMPOSE_ROOT/script/test/fixtures/flask-redis/output-k8s.json" "Unsupported key depends_on - ignoring"
-# openshift test
-convert::expect_success_and_warning "kompose convert --stdout --dc -f $KOMPOSE_ROOT/script/test/fixtures/flask-redis/docker-compose.yml" "$KOMPOSE_ROOT/script/test/fixtures/flask-redis/output-os.json" "Unsupported key depends_on - ignoring"
-
-######
```
Comment (surajssd) [inline script/test/cmd/tests.sh]: Removed similar tests.

File: script/test/fixtures/etherpad/output-k8s.json [hunk 3]
```diff
@@ -7,48 +7,44 @@
       "kind": "Deployment",
       "apiVersion": "extensions/v1beta1",
       "metadata": {
-        "name": "etherpad",
-        "creationTimestamp": null,
+        "name": "mariadb",
+        "creationTimestamp": null
```
Comment (surajssd) [inline script/test/fixtures/etherpad/output-k8s.json]: Updated test fixtures on current master.

**Comment by kadel** [comment]
LGTM

**Comment by surajssd** [comment]
Merging. Thanks @kadel for review.

**Pull request merged**